src/main/NWN/API/Object/NwArea.cs

```diff
@@ -216,5 +216,21 @@ public void SetFogColor(FogType fogType, FogColor fogColor)
     /// </summary>
     /// <returns>The result of this destroy action.</returns>
     public AreaDestroyResult Destroy() => (AreaDestroyResult) NWScript.DestroyArea(this);
+
+    /// <summary>
+    /// Sets the main light colors on the tile located at (tileLocation).
+    /// (tile) specifies the location of the tile.
+    /// You must call RecomputeStaticLighting() after calling this function for changes to occur for the players.
+    /// </summary>
+    public void TileMainLightColor(Location tileLocation, TileMainLightColor first, TileMainLightColor second)
+      => NWScript.SetTileMainLightColor(tileLocation, (int)first, (int)second);
+
+    /// <summary>
+    /// Sets the source light colors on the tile located at (tileLocation).
+    /// (tile) specifies the location of the tile.
+    /// You must call RecomputeStaticLighting() after calling this function for changes to occur for the players.
+    /// </summary>
+    public void TileSourceLightColor(Location tileLocation, TileSourceLightColor first, TileSourceLightColor second)
+      => NWScript.SetTileMainLightColor(tileLocation, (int)first, (int)second);
```

Comment:
Move this to Location.Managed next to the getters.

Reply:
Done.